Resources/Prototypes/Entities/Structures/Machines/microwave.yml

```diff
@@ -83,6 +83,7 @@
       behaviors:
       - !type:DoActsBehavior
         acts: ["Breakage"]
+      - !type:ExplodeBehavior
```

Comment:
Do we really want any damage to a microwave to cause it to explode, even just someone punching it really hard?

Reply:
if you meant to make the microwave explosive, like i suggested in my review, you just have to add the ExplosiveComponent. The explode behavior is only for if you want it on destruction (no, imo)